21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more
21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more

21 Nov 2017 11:34:30,545 ERROR it.eng.spagobi.rest.interceptors.RestExceptionMapper.toResponseFromGenericException:97 - Catched service error:
it.eng.spagobi.utilities.exceptions.SpagoBIRestServiceException: generic.error
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:912)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
    at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
    at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
    at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:542)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:524)
    at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:126)
    at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
    at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
    at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:409)
    at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1044)
    at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:607)
    at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:315)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:748)
Caused by: it.eng.spagobi.utilities.exceptions.SpagoBIServiceException: An unexpected error occured while executing service
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:771)
    at it.eng.spagobi.api.v2.DataSetResource.getDataStorePost(DataSetResource.java:910)
    ... 34 more
Caused by: java.lang.RuntimeException: An unexpected error occured while executing method
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:456)
    at it.eng.spagobi.api.DataSetResource.getDataStore(DataSetResource.java:755)
    ... 35 more
Caused by: it.eng.spagobi.tools.dataset.cache.CacheException: An unexpected error occured while persisting store in cache
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1115)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.put(SQLDBCache.java:1028)
    at it.eng.spagobi.tools.dataset.DatasetManagementAPI.getDataStore(DatasetManagementAPI.java:425)
    ... 36 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more
Caused by: org.postgresql.util.PSQLException: ERROR: length for type varchar cannot exceed 10485760
  Position: 121
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:555)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:403)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:395)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.executeStatement(PersistedTableManager.java:595)
    at it.eng.spagobi.tools.dataset.persist.PersistedTableManager.createTable(PersistedTableManager.java:814)
    at it.eng.spagobi.tools.dataset.cache.impl.sqldbcache.SQLDBCache.persist(SQLDBCache.java:1073)
    ... 38 more